annotator-core/src/test/java/edu/ucr/cs/riple/core/DownstreamAnalysisTest.java

```diff
@@ -100,16 +100,17 @@ public void lowerBoundComputationTest() {
   public void upperBoundComputationTest() {
     coreTestHelper
         .addExpectedReports(
-            // Only returnNullableBad triggers new errors in this fix chain (+9) and upper bound
-            // should be 9
-            new TReport(new OnMethod("Foo.java", "test.target.Foo", "returnNullableBad(int)"), 9),
+            // Only returnNullableBad triggers new errors in this fix chain (+10) and upper bound
+            // should be 10
+            new TReport(new OnMethod("Foo.java", "test.target.Foo", "returnNullableBad(int)"), 10),
             // Only returnNullableGood triggers new errors in this fix chain (+1) and upper bound
             // should be 1
             new TReport(new OnMethod("Foo.java", "test.target.Foo", "returnNullableGood(int)"), 1),
             // Root fix triggers 1 error on downstream dependency and returnNullableBad is
-            // present in the fix tree and triggers 9 errors on downstream dependency, therefore the
-            // upper bound effect for the tree should be 10.
-            new TReport(new OnMethod("Foo.java", "test.target.Foo", "bar()"), 10))
+            // present in the fix tree and triggers 10 errors on downstream dependency, therefore
+            // the
+            // upper bound effect for the tree should be 11.
```

Comment:
bad formatting

Reply:
Fixed in f1598b7db98d7884869ab8ee7d41e5c291e98dd5